### PR TITLE
Allow setting heartbeat via props.

### PR DESCRIPTION
**Pull request by wedgemartin**

### CODE REVIEW

**Comment by lahsivjar** [comment]
@wedgemartin LGTM. Is it ready to merge?

**Comment by wedgemartin** [comment]
Good deal.  Sorry I accidentally added the dist to the repo.  I removed and added the PropType values for the heartbeat params and made a change to check if the value is set via Object.keys so that 0 doesn't get ignored as a value.

File: src/client.jsx [hunk 1]
```diff
@@ -73,6 +76,16 @@ class SockJsClient extends React.Component {
   _initStompClient = () => {
     // Websocket held by stompjs can be opened only once
     this.client = Stomp.over(new SockJS(this.props.url));
+    if (this.props.heartbeat) {
+      this.client.heartbeat.outgoing = this.props.heartbeat;
+      this.client.heartbeat.incoming = this.props.heartbeat;
+    }
+    if (Object.keys(this.props).includes('heartbeatIncoming')) {
```
Comment (lahsivjar) [inline src/client.jsx]: @wedgemartin Let's define the default values in `defaultProps` and remove the if else entirely. We can use the defaults provided by stomjs for it (10000ms)

**Comment by lahsivjar** [comment]
@wedgemartin Also it would be great if you can squash the commits into a single commit.

**Pull request closed without merging**